buildSrc/src/main/kotlin/scripts/variants.gradle.kts

```diff
@@ -107,5 +119,68 @@ android {
             val activated = FeatureFlags.activated[flavor.name].orEmpty().contains(feature)
             flavor.buildConfigField("Boolean", feature.name, activated.toString())
         }
+
+        Configs.values().forEach { configs ->
+            when (configs.configType) {
+                ConfigType.STRING -> {
+                    flavor.buildConfigField(
+                        configs.configType.type, configs.name, """"${buildtimeConfiguration?.configuration?.get(configs.value)}""""
+                    )
+                }
+                ConfigType.INT, ConfigType.BOOLEAN -> {
+                    flavor.buildConfigField(
+                        configs.configType.type, configs.name, buildtimeConfiguration?.configuration?.get(configs.value).toString()
+                    )
+                }
```

Comment:
Isn't the same thing, can't we do for both: 
`""""${buildtimeConfiguration?.configuration?.get(configs.value).toString()}""""`